apps/music/views/albums/view.js

```diff
@@ -77,6 +77,7 @@ AlbumsView.prototype.getAlbums = function() {
         .then(response => response.json())
         .then((albums) => {
           albums.forEach((album) => {
```

Comment:
Also, instead of copying the full metadata object over to the new one, just grab `album` and `artist`. This may end up being somewhat of a mini-megapatch, but it'll be worth it.